test/conformance/helpers/tracing/zipkin.go

```diff
@@ -47,9 +50,14 @@ var setTracingConfigOnce = sync.Once{}
 // TODO Do we need a tear down method to revert the config map to its original state?
 func setTracingConfigToZipkin(t *testing.T, client *lib.Client) {
 	setTracingConfigOnce.Do(func() {
+		// Get the namespace from the env key directly as system.Namespace() panics if unset
+		ns := os.Getenv(system.NamespaceEnvKey)
+		if ns == "" {
+			ns = "istio-system"
```

Comment:
Is this code path tested?

Reply:
I'm not too sure how coverage for any of this code is provided, I'd be happy to add coverage to any tests that currently exist to exercise this new path. Thanks!